src/runtime/motion/spring.ts

```diff
@@ -110,22 +114,31 @@ export function spring<T=any>(value?: T, opts: SpringOpts = {}): Spring<T> {
 
 				inv_mass = Math.min(inv_mass + inv_mass_recovery_rate, 1);
 
+				last_time = Math.max(last_time, now - tick * limit);
+				while ( last_time < now ) {
```

Comment:
please indent everything inside the `while` loop appropriately
```suggestion
				while (last_time < now) {
```